packages/api/types.ts

```diff
@@ -1782,3 +1782,109 @@ export type TSocketPaginateableEndpointNames = KeysMatching<
     TSocketEndpoints,
     { request: { limit?: number; offset?: number } }
 >;
+
+interface ExtendedMT5AccountsListResponse {
+    mt5_login_list?: ExtendedMt5LoginList;
+
+    echo_req: {
+        [k: string]: unknown;
+    };
+
+    msg_type: 'mt5_login_list';
+
+    req_id?: number;
+    [k: string]: unknown;
+}
+
+type ExtendedMt5LoginList = ExtendedDetailsOfEachMT5Loginid[];
+
+type TEligibleToMigrate = {
+    synthetic: string;
+    financial: string;
+};
+
+interface ExtendedDetailsOfEachMT5Loginid {
+    account_type?: 'demo' | 'real';
+
+    balance?: number;
+
+    country?: string;
+
```

Comment:
what are the new lines for? 😄

Reply:
Hey..the new parameters are not yet added to the response (as be has not released it yet). This, was causing build to fail so we added this new type as a temporary fix. Will remove this once the schema is updated.